src/Application/Configuration.php

```diff
@@ -53,4 +63,9 @@ public function getVersion(): string
     {
         return $this->version;
     }
+
+    public function getSource(): ?string
```

Comment:
I don't see this method being called anywhere. I think you should remove it

Reply:
Hi Marcelo, this method is used in the BaseManager inside the method generateRequestHeaders, to set the headers:
https://github.com/LinioIT/seller-center-sdk/pull/94/files#diff-d1a09b8202dddee91bcfa84e6dcc7960291dcd27b8458092a3402ce3f547da32R80